LICENSE

```diff
@@ -265,3 +265,14 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+---------------
+
+pekko-http-core contains code based on code by Tim Kienzle with his permission to use it under an Apache License
+(see https://github.com/apache/incubator-pekko-http/issues/195#issuecomment-1610633609).
```

Comment:
Can the response be protected from deletion?

Reply:
This is hopefully a link that cannot be removed - https://lists.apache.org/thread/v61twg09h0spcdzk2v8hb5cccs2krn12

Reply:
Thanks, changed the link to that one. In any case, this is a low risk situation, so let's not overthink it.